packages/appium/lib/config.js

```diff
@@ -213,8 +215,6 @@ function validateServerArgs (parser, args) {
   }
 
   const validations = {
-    port: checkValidPort,
```

Comment:
why these validators are removed?

Reply:
they are handled by the schema itself, and validation of the arguments thru `ajv`

Reply:
those other ones that I _didn't_ remove... I have no idea what to do with those, because they aren't in `--help`